src/app/modules/shared-components/components/score-ring/score-ring.component.ts

```diff
@@ -1,7 +1,7 @@
 import { Component, Input, OnChanges, SimpleChanges, ViewChild, ElementRef } from '@angular/core';
 
 enum ScoreRingColor {
-  Success = '#B8E986',
+  Success = 'rgb(105, 206, 77)',
```

Comment:
I wonder if we should use the equivalent hex notation to keep consistency with the other colors below, your call

Reply:
Hex Fixed

Reply:
Yes as component input is ok, I think by default it should be adapted to a light background because it's the most common use case

Reply:
Fixed